docs/_docs/themes.md

```diff
@@ -4,28 +4,37 @@ title: Themes
 permalink: /docs/themes/
 ---
 
-Jekyll has an extensive theme system, which allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package layouts, includes, and stylesheets in a way that can be overridden by your site's content.
+Jekyll has an extensive theme system that allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package layouts, includes, and stylesheets in a way that can be overridden by your site's content.
 
-## Installing a theme
+## Understanding gem-based themes
 
-1. To install a theme, first, add the theme to your site's `Gemfile`:
+When you [create a new Jekyll site](/docs/quickstart) (by running the `jekyll new <PATH>` command), Jekyll installs a gem-based theme called [Minima](https://github.com/jekyll/minima). 
 
-        gem 'my-awesome-jekyll-theme'
+With [gem-based themes](/docs/themes/), some of the theme directories and files are stored in the gem, hidden from view in your Jekyll project. As a result, the files and directories shown for your site are a subset of all the theme's files. In the case of Minima, you see only the following:
```

Comment:
> the files and directories shown for your site are a subset of all the theme's files

Not a subset. There are some files in my theme, there are other files in my site. I don't think the venn diagram is as this sentence makes it out to be.